datasets/modeled/transport/dataset.json5

```diff
@@ -0,0 +1,110 @@
+{
+  dataset_id: "tempo_conus_2022",
```

Comment:
@ahcyip I copied over some information from the StandardScenarios version of TEMPO and made some minor adjustments. Please edit as necessary and add dimension and mapping files.